release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2678,6 +2677,29 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.15.8
+[id="ocp-4-15-8"]
+=== RHSA-2024:1668 - {product-title} 4.15.8 bug fix and security update
+
+Issued: 2024-04-08
+
+{product-title} release 4.15.8, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:1668[RHSA-2024:1668] advisory.
```

Comment:
Was there no RPM for this release? I think in the past we have had a sentence stating there was no RPMs for the release. Disregard if you all have decided not to do this since I left RNs.

Reply:
Here is a link to a super old example https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-57

Reply:
Good point. I was not sure how to handle this rare scenario. Updated!